corehq/apps/app_manager/suite_xml/sections/endpoints.py

```diff
@@ -72,8 +73,14 @@ def _make_session_endpoint(self, module, form=None):
             stack=stack,
         )
 
-    def _get_frame_children(self, module, form):
-        helper = WorkflowHelper(self.suite, self.app, self.modules)
+
+class EndpointsHelper(object):
```

Comment:
OOC, why make this a class rather than a function?

Reply:
just mimicking the setup of other helper classes